From 10e5ea6d3390b4dcfd7eea8a54b3396ba222cb5a Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Fri, 26 Aug 2016 13:48:55 -0700
Subject: [PATCH] OS-5634 system-log SMF method has existential confusion
 Reviewed by: Alex Wilson <alex.wilson@joyent.com> Reviewed by: Josh Wilsdon
 <jwilsdon@joyent.com>

---
 overlay/generic/lib/svc/method/system-log | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)
 mode change 100644 => 100755 overlay/generic/lib/svc/method/system-log

diff --git a/overlay/generic/lib/svc/method/system-log b/overlay/generic/lib/svc/method/system-log
old mode 100644
new mode 100755
index f8a60a36..25f67919
--- a/overlay/generic/lib/svc/method/system-log
+++ b/overlay/generic/lib/svc/method/system-log
@@ -103,20 +103,21 @@ fi
 
 if [ -f /etc/rsyslog.conf ]; then
 	if [ ! -x /usr/sbin/rsyslogd ]; then
-		echo "/etc/rsyslog.conf is missing. Exiting."
+		echo "Executable /usr/sbin/rsyslogd not found or not executable. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 else
 	#
 	# Fall back to old syslogd
 	#
+	echo "/etc/rsyslog.conf is missing. Using syslogd instead."
 	if [ ! -f /etc/syslog.conf ]; then
 		echo "/etc/syslog.conf is missing. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 
 	if [ ! -x /usr/sbin/syslogd ]; then
-		echo "Executable /usr/sbin/syslogd not found. Exiting"
+		echo "Executable /usr/sbin/syslogd not found or not executable. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 fi
-- 
2.21.0

